pennylane/templates/subroutines/qsvt.py

```diff
@@ -285,10 +285,12 @@ def __init__(self, UA, projectors, id=None):
             "projectors": projectors,
         }
 
-        ua_wires = UA.wires.toset()
-        proj_wires = set.union(*(proj.wires.toset() for proj in projectors))
+        ua_wires = qml.wires.Wires(UA.wires)
```

Comment:
Could it be more compact? e.g., as

`total_wires = UA.wires + sum(proj.wires for proj in projectors)`

Reply:
For some reason, doing that adds the wires 0 even if I'm not using it. 🤔

Reply:
solved